src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -138,7 +140,44 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
   }
 
   def deleteResource(resource: FullyQualifiedResourceId): IO[Unit] = ???
-  def loadResourceAuthDomain(resource: FullyQualifiedResourceId): IO[LoadResourceAuthDomainResult] = ???
+
+  def loadResourceAuthDomain(resource: FullyQualifiedResourceId): IO[LoadResourceAuthDomainResult] = {
+    runInTransaction { implicit session =>
+      val ad = AuthDomainTable.syntax("ad")
+      val r = ResourceTable.syntax("r")
+      val rt = ResourceTypeTable.syntax("rt")
+      val g = GroupTable.syntax("g")
+
+      val query = samsql"""select ${g.result.name}
+              from ${ResourceTable as r}
+              join ${ResourceTypeTable as rt} on ${r.resourceTypeId} = ${rt.id}
+              left outer join ${AuthDomainTable as ad} on ${r.id} = ${ad.resourceId}
+              left outer join ${GroupTable as g} on  ${ad.groupId} = ${g.id}
+              where ${r.name} = ${resource.resourceId} and ${rt.name} = ${resource.resourceTypeName}"""
```

Comment:
I didn't use a pkQuery here and just wrote out the joins because that what made sense to me